backend/document-service/src/main/kotlin/com/nosql/document/dto/ContactsDto.kt

```diff
@@ -0,0 +1,18 @@
+package com.nosql.document.dto
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.annotation.JsonProperty
+import javax.validation.constraints.Email
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonIgnoreProperties(ignoreUnknown = true)
+class ContactsDto (
```

Comment:
use `data class`

Reply:
this make no sense, we do not use toString